arroyo-sql/src/expressions.rs

```diff
@@ -2041,8 +2085,7 @@ impl StringFunction {
                     })
                 }
             }
-            StringFunction::RegexpMatch(_, _) => todo!(),
-            StringFunction::RegexpReplace(_, _, _, _, _) => todo!(),
+            StringFunction::RegexpReplace(_, _, _, _) => unreachable!(),
```

Comment:
I'd clarify this is the `Some(_)` in the fourth argument case here.

Reply:
Fixed with https://github.com/ArroyoSystems/arroyo/pull/122/commits/9889c37a41d5d3ae6f678c500f5cbcacf754ad9e